src/index.tsx

```diff
@@ -130,6 +130,19 @@ export function readBucketedRecords(
   recordType: RecordType,
   options: BucketedRequestOptions
 ): Promise<BucketedRecordsResult> {
+  // These types aren't currently supported by the aggregateByDuration sdk method
+  // TODO: Handle blood pressure aggregate only available on Android 15+
+  if (
+    [
+      'BloodPressure',
+      'BodyTemperature',
+      'HeartRateVariabilityRmssd',
+      'SleepSession',
+    ].includes(recordType)
+  ) {
+    return HealthConnect.readManuallyBucketedRecords(recordType, options);
+  }
```

Comment:
Just wanted to check whether there was something to do in this PR or whether it'll be addressed separately?

Reply:
I don't think we currently target android 15 yet so using the android aggregate query would need to be done after then - so yes will be address separately and the comment is more of a reminder for that point. Unless there was something else you were referring to?